src/arch/generic/semihosting.cc

```diff
@@ -649,18 +649,26 @@ BaseSemihosting::FileBase::flen()
     return -EINVAL;
 }
 
+static const std::vector<uint8_t> features;
```

Comment:
I am a bit confused. How is this used?

Reply:
Oops that accidentally slipped into the uploaded patch
```suggestion
```